packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
I didn't check the spec, but it seems like from the [compat-table](https://kangax.github.io/compat-table/es6/) that firefox doesn't have default function paramters implemented all the way (tdz, etc)